db/upgrade.php

```diff
@@ -64,6 +64,43 @@ function create_xapi_log_table($dbman, $tablename) {
     }
 }
 
+/**
+ * Create new columns in the database.
+ * 
+ * @param object $dbman
+ * @param string $tablename
+ *
+ */
+function add_logstorestandardlogid_type_to_table($dbman, $tablename) {
+    // Select table.
+    $table = new xmldb_table($tablename);
+
+    // Conditionally add field logstorestandardlogid to logstore_xapi_log.
```

Comment:
Comment needs an update as it mentions a specific table.
The same applies on line 85

Reply:
I have fixed this.